scratchpad/qaoa_eng_vs_backends.py

```diff
@@ -272,7 +272,12 @@ def cook_raw_report(raw_reports, task_type = "QAOAEnergyExpectation"):
 if __name__ == '__main__':
     
     # Should I use any or all?
-    raw_reports = qaoa_energy_benchmarking(repeat = 7)
-    cooked_reports = cook_raw_report(raw_reports)
-    with open("sample.json", "w") as outfile:
-        json.dump(cooked_reports, outfile, indent = 4, sort_keys= True)
```

Comment:
The benchmarking script should print json, not write it to file. If anyone wants to write it, they can just pipe it with ` > file.jsonl`.